### PR TITLE
 - Display Group Profile Titles on Info page instead of Group Slugs

### PR DESCRIPTION
**Pull request by afabiani**

### CODE REVIEW

File: geonode/documents/views.py [hunk 1]
```diff
@@ -113,12 +113,19 @@ def document_detail(request, docid):
...
+                group = GroupProfile.objects.get(slug=document.group.name)
+            except GroupProfile.DoesNotExist:
+                group = None
         context_dict = {
             'perms_list': get_perms(
                 request.user,
                 document.get_self_resource()),
             'permissions_json': _perms_info_json(document),
             'resource': document,
+            'group': group,
```
Comment (capooti) [inline geonode/documents/views.py]: I don't get why you want to add group in the context dict. It should be always possible to get it from resource.group in the template

Comment (afabiani) [reply geonode/documents/views.py]: It is not the group but the group profile associated to it

Comment (afabiani) [reply geonode/documents/views.py]: It is not the group but the group profile associated to it

Comment (afabiani) [reply geonode/documents/views.py]: @capooti unless further clarifications, if you are ok with the solution proposed, can you approve the PR again? Thanks.

Comment (capooti) [reply geonode/documents/views.py]: oh I see, I thought geonode.Group had a FK with django.auth.Group (as for geonode.Profile)

**Pull request merged**